frontend/src/routes.tsx

```diff
@@ -194,7 +195,7 @@ export const AppRoutes = (): JSX.Element => {
 						</PrivateRoute>
 					}
 				/>
-				<Route path="/public/files/:fileId" element={<PublicFileComponent />} />
+				<Route path="/public_files/:fileId" element={<PublicFileComponent />} />
```

Comment:
Are we using public/files or public_files? 
In other PRs that got merged in, I fixed places that are using "_"...

Reply:
If you would like to use "_", you might have to 1. merge the latest main, 2. do a project wide search of places still using /
E.g. If you go to a file under public dataset, it right now still goes to /public/file and hence broken.

Reply:
I think I got all of these fixed with a merge and a search/replace. Testing.

Reply:
I also think that files should have a status. Checking the populate fake data, the status for files is 'null' in some cases and those should match the datasets.

Reply:
Thank you. This has been resolved.